packages/pwa/app/pages/checkout/partials/contact-info.test.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {screen, within} from '@testing-library/react'
+import user from '@testing-library/user-event'
+
+import ContactInfo from './contact-info'
+import {renderWithProviders} from '../../../utils/test-utils'
+
+jest.mock('../util/checkout-context', () => {
+    return {
+        useCheckout: jest.fn().mockReturnValue({
+            customer: null,
+            basket: {},
+            isGuestCheckout: true,
+            setIsGuestCheckout: jest.fn(),
+            step: 0,
+            login: null,
+            setCheckoutStep: null,
+            goToNextStep: null
+        })
+    }
+})
+
+jest.mock('../../../commerce-api/hooks/useCustomer')
```

Comment:
Seems like we can delete this mock. When I did that, the test still passes.

Reply:
done